### ISSUE TITLE
Request file

### ISSUE THREAD

**Issue by Anita0124** [zouguojian/traffic-demand-prediction#1]
Hello, Mr. Zou, thank you very much for the running code you provided on Github. I have a question here and I want to ask you. The  test.py reports "No such file or directory: 'data/adjacent.csv" during running. May I ask Is the file not uploaded?
Thanks above.

**Comment by zouguojian** [comment]
Yes.Because my &nbsp;paper is being reviewed for review. As long as the paper is accepted at the first time, I will open up the data, thank you!



------------------ Original ------------------
From: Anita ***@***.***&gt;
Date: Sun,Nov 21,2021 9:42 AM
To: zouguojian/traffic-demand-prediction ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [zouguojian/traffic-demand-prediction] Request file (Issue #1)





 
Hello, Mr. Zou, thank you very much for the running code you provided on Github. I have a question here and I want to ask you. The  test.py reports "No such file or directory: 'data/adjacent.csv" during running. May I ask Is the file not uploaded?
 Thanks above.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Anita0124** [comment]
> Yes.Because my &nbsp;paper is being reviewed for review. As long as the paper is accepted at the first time, I will open up the data, thank you!
> […](#)
> ------------------ Original ------------------ From: Anita ***@***.***&gt; Date: Sun,Nov 21,2021 9:42 AM To: zouguojian/traffic-demand-prediction ***@***.***&gt; Cc: Subscribed ***@***.***&gt; Subject: Re: [zouguojian/traffic-demand-prediction] Request file (Issue #1) Hello, Mr. Zou, thank you very much for the running code you provided on Github. I have a question here and I want to ask you. The test.py reports "No such file or directory: 'data/adjacent.csv" during running. May I ask Is the file not uploaded? Thanks above. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

Thank again. here is a little request. Is it convenient to provide versions of Python, Tensorflow and Keras in which you run your code?

**Comment by zouguojian** [comment]
> 

I can tell you. But the premise is, I want to ask, what is your intention to use my thesis code. Because I am worried about the idea of the thesis leaking. Thank you for your understanding.
